### ISSUE TITLE
Adding "on demand" loading of a poster preview

### ISSUE THREAD

**Issue by yarikoptic** [datalad-datasets/ohbm2020-posters#37]
now that we have access to PDFs (I will soon populate git-annex of this repo with them) I wondered if it would be of any benefit to generate preview pngs for all of them, and then may be make them load/shown somehow, e.g. have a PREVIEW icon next to PDF with JS which would load up and show that png when hovered over, and disappear when moved away.

I am not feeling strongly about benefit of such functionality -- just want to throw against the wall if someone might find it interesting/useful

**Comment by soichih** [comment]
I agree thumbnails themselves might not give much info.. but I think it could help to create an atmosphere of being the real poster session? I am also a very visual person.. so for me thumbnails would probably help me to locate posters that I've seen before..

So I would give +1 to this!

**Comment by yarikoptic** [comment]
+100 on atmosphere. Someone could probably even make a carousel above or below the table for the present in the table currently posters and bring out a bit bigger size for the hovered over poster?

**Comment by soichih** [comment]
I've created Nx400 pixel thumbs of all posters. 1835 PNGs, around 300MB total. I think we can put them all under assets.. will it be a problem? I think I can reduce the total size down to 100MB-200MB if we use jpg and/or reduce the resolution.

I am thinking.. maybe we could lazy-load the thumbs on all records with reduced sizes (Nx150) and let user hover over to show the Nx400 size images. What do you all think?

**Comment by yarikoptic** [comment]
We mustn't commit them to git here -- too big. What was your script/naming? Let's add to  a separate dedicated GitHub repo (later linked here as a subdataset) - I will create a new repo shortly for you to push into.
 I will finally get to work on adding PDFs to annex now, and then make those thumbnails reproducible...

**Comment by yarikoptic** [comment]
https://github.com/datalad-datasets/ohbm2020-posters-thumbnails you had been invited @soichih
